docs/contributors/getting-started.md

```diff
@@ -84,6 +84,34 @@ but most popular are:
 - debugging through logging (recommended option)
 - classic debugging with breakpoints
 
+## Commit messages
+
+Try to follow the
+[conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/),
+which means your commits should be of form:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Additionally, most should be explained in order for reviewers to understand it
```

Comment:
It's not really clear, where you want this explanation to go. I assume it should be in the PR description (?).

Reply:
I changed based on @ckipp01 's feedback, is it a bit clearer now? I assume that in one-commit PR the messages will be reused between the commit and the PR. This might be different in some other cases. I mention that below.